lib/tasks/occurred_at.rake

```diff
@@ -0,0 +1,11 @@
+namespace :incidents do
+  desc 'move occurred_at to incidents'
+  task move_occurred_at: :environment do
+    Incident.all.each do |incident|
+      if incident.driver_incident_report?
+        occurred_at = incident.driver_incident_report.occurred_at
+        incident.update(occurred_at: occurred_at)
```

Comment:
With a task that updates _every object_ in the database like this, there's a (slight) concern for me that for some reason past incidents won't be valid for other reasons. This has happened before. So updating it won't work in that case. Do we want to skip validations here?